docs/markers.md

```diff
@@ -4,15 +4,21 @@ This section is a work-in-progress. Markers are events that happen within the br
 
 ## Implementing new markers
 
-Markers are implemented in [ProfilerMarkers.cpp] and [ProfilerMarkers.h]. There are C++ macros for adding instrumentation in different parts of the codebase. These are defined in [GeckoProfiler.h] and are [`PROFILER_MARKER`][PROFILER_MARKER] and [`PROFILER_MARKER_PAYLOAD`][PROFILER_MARKER_PAYLOAD]. Markers without a payload only have a single start time. Markers with a payload can have a start and end time, as well as additional information. The payloads are defined in [ProfilerMarkers.h] and [ProfilerMarkers.cpp].
-
-[ProfilerMarkers.cpp]: http://searchfox.org/mozilla-central/source/tools/profiler/core/ProfilerMarkers.cpp
-[ProfilerMarkers.h]: http://searchfox.org/mozilla-central/source/tools/profiler/public/ProfilerMarkers.h
+Markers are implemented in [ProfilerMarker.h], [ProfilerMarkerPayload.h] and
+[ProfilerMarkerPayload.cpp].
+Markers are added to a profile with the [`profiler_add_marker'] function.
```

Comment:
Did you want to mark the function name between backticks ? Then the last tick is wrong and makes github crazy :)

But I think we don't need the ticks because the link will make the name stand out already. Would you please fix this last issue ?